examples/mockups/index.html

```diff
@@ -146,23 +146,23 @@ <h2 class="ui header">How it works</h2>
                     <img src="images/infographic-01.png">
                   </div>
                   <p>
-                    Write your documentation easily, using .md or .rst, directly in your own IDE, as you work on what maters: your project.
+                    Write your documentation easily, using .md or .rst, directly in your own IDE, as you work on what matters: your project.
                   </p>
                 </div>
                 <div class="column center aligned">
                   <div class="ui image">
                     <img src="images/infographic-02.png">
                   </div>
                   <p>
-                    Upload your work to your favorite source code repository hosting service. We support GitHub, GitLab, BitBucket and more.
+                    Upload your work to your favorite code hosting service. Use GitHub, GitLab, BitBucket, or your own.
                   </p>
                 </div>
                 <div class="column center aligned">
                   <div class="ui image">
                     <img src="images/infographic-03.png">
                   </div>
                   <p>
-                    Setup your account with Read the Docs, add a .config file to tell us your preferences and we'll do the rest!
+                    Setup your account with Read the Docs, add a <code>.readthedocs.yaml</code> file to set the configuration, and we'll do the rest!
```

Comment:
Wrote the actual config file name. Should we point to our docs here? https://docs.readthedocs.io/en/stable/config-file/v2.html

Reply:
The changes look great.

I'm of two minds about adding the link; in one hand, that resource is not the start point for a new user, so by sending him there, we can be diverging him, on the other hand, if we think that someone comes to this text looking information on resources, than the link should be there.

Reply:
I'd agree we're probably distracting the user at this point by pointing them to the docs on our config file -- especially because those docs are *dense* technical docs. For users that aren't familiar with RTD, they can get a sense of things through our guides.